engine/Shopware/Controllers/Backend/Order.php

```diff
@@ -729,9 +754,22 @@ public function batchProcessAction()
             $data['paymentStatus'] = Shopware()->Models()->toArray($order->getPaymentStatus());
             $data['orderStatus'] = Shopware()->Models()->toArray($order->getOrderStatus());
 
-            $data['mail'] = $this->checkOrderStatus($order, $statusBefore, $clearedBefore, $autoSend, $documentType, $addAttachments);
-            //return the modified data array.
-            $orders[$key] = $data;
+            try {
+                // The method '$this->checkOrderStatus()' (even its name would not imply that) sends mails and can fail
```

Comment:
Please remove the passive-aggressive text in parentheses and re-wrap the comment 😉